packages/frontend/src/view/pages/user/components/UserAssetTable.tsx

```diff
@@ -93,3 +102,8 @@ export function UserAssetsTable(props: UserAssetsTableProps) {
     />
   )
 }
+
+export function getEscapeHatchLink(
+  vaultOrPositionId: string,
+  ethereumAddress: EthereumAddress
+) {}
```

Comment:
This implementation seems surprisingly short :)

Reply:
hahah, true